client/state/ui/guided-tours/selectors/index.js

```diff
@@ -22,7 +22,8 @@ import debugFactory from 'debug';
  * Internal dependencies
  */
 import { GUIDED_TOUR_UPDATE, ROUTE_SET } from 'state/action-types';
-import { getInitialQueryArguments, getSectionName } from 'state/ui/selectors';
+import { getSectionName } from 'state/ui/selectors';
+import getInitialQueryArguments from 'state/selectors/get-initial-query-arguments';
```

Comment:
Should be `getInitialQueryArguments from '{ state/selectors }`.

But I really kinda distrust these selectors altogether. What does initial vs current qargs mean? That just adds confusion to something that's confusing enough IMO.

Reply:
Asked for motivation: https://github.com/Automattic/wp-calypso/pull/6417#issuecomment-343751609